app/routes/content.js

```diff
@@ -104,8 +104,17 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated')).toISOString()],
-                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()]
+                ['article:published_time', d.toISOString()],
+                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()],
+                ['citation_title', node.get('title')],
+                ['citation_description', node.get('description')],
+                ['citation_public_url', window.location.href],
```

Comment:
The API response should have a web url for this preprint, right? That seems a better choice than the location.